packages/react/src/components/navigation/progress/Stepper/index.tsx

```diff
@@ -133,13 +143,12 @@ const stepContentsByState = {
 
 export const Step = memo(function Step({
   state,
-  label,
-  hideLeftSeparator,
+  label: Label,
+  hideLeftSeparator = true,
```

Comment:
Wondering why this change (defaults to `true`) here?

Reply:
Good catch that's a mistake, will change that